lib/elixir_analyzer/test_suite/newsletter.ex

```diff
@@ -0,0 +1,89 @@
+defmodule ElixirAnalyzer.TestSuite.Newsletter do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Newsletter 
+  """
+
+  alias ElixirAnalyzer.Constants
+
+  use ElixirAnalyzer.ExerciseTest
+
+  feature "close_log ends with File.close" do
+    type :actionable
+    comment Constants.newsletter_close_log_returns_implicitly()
+
+    form do
+      def close_log(_ignore) do
+        _block_ends_with do
+          File.close(_ignore)
+        end
```

Comment:
Nice how useful the new feature is 🙂 👏

It will cause a false-positive if somebody uses `IO.inspect` as the last call or assigns the return value to a variable and puts that variable as the last expression. But both are bad style so I don't care. Debug calls to `IO.inspect` should be removed before submitting a solution and the `x = File.close(x); x` is a compilation warning about a redundant variable.

I was thinking about adding a common check for `IO.inspect` in all exercises but a few that actually require it (rpg character sheet I think). To prompt people to clean up the debug calls. What do you think?

Reply:
> I was thinking about adding a common check for `IO.inspect` in all exercises but a few that actually require it (rpg character sheet I think). To prompt people to clean up the debug calls. What do you think?

Yes, I'm interested in working on that. I'll have to think of a clean way do exclude a common check. First thing that popped into my mind is:
```elixir
defmodule ElixirAnalyzer.TestSuite.RpgCharacterSheet do
  use ElixirAnalyzer.ExerciseTest,
    suppress_check: ElixirAnalyzer.Constants .solution_debug_functions(),
...
end
```
I would then pool all the test (feature/assert_call/common) in `ExerciseTest` and filter out the suppressed ones. It would also allow a `feature` to use `suppress_if` in reference to an `assert_call` check, which I have been thinking about doing.

Reply:
>  It would also allow a feature to use suppress_if in reference to an assert_call check, which I have been thinking about doing.

That's an excellent idea, I noticed that that's missing too...

I created a ticket for this as a reminder 🙂